### PR TITLE
Add base address of PYUSD

### PR DESCRIPTION
**Pull request by skellet0r**

### CODE REVIEW

**Comment by roberto-bayardo** [comment]
Unfortunately the current factory doesn't support custom decimal amounts.

```
error: PYUSD on chain base token 0x5fd0787a5A66D24C5a868c2e6a296ca2B1452558 has incorrect decimals
```

if you can hang tight for a bit, @hamdiallam should be sharing a new factory implementation soon that does allow custom decimals, which you can use to de-deploy the L2 token.

**Comment by hamdiallam** [comment]
+1 will share a temp solution soon

**Comment by hamdiallam** [comment]
Hey @skellet0r, the factory and OptimismMintableERC20 contracts in the develop branch of ethereum-optimism/optimism was just updated such that decimals can be sepecified. Here's the associated PR https://github.com/ethereum-optimism/optimism/pull/7182.

I would recommend getting a deployment of the factory up yourself to use to generate the right L2 token. Eventually we'll have our factories updated

File: data/PYUSD/data.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "PayPal USD",
+  "symbol": "PYUSD",
+  "decimals": 6,
```
Comment (roninjin10) [inline data/PYUSD/data.json:4]: hey @skellet0r let me know if you need any help updating this with the new factory

**Review by roninjin10 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**